{"timestamp":"2025-09-22T16:30:39.181909","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-22T16:30:39.187692","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/API_POSTGRE.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-22T16:30:46.630431Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.633691Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.634129Z","level":"info","event":"Current task name:load_csv_to_stg","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.634457Z","level":"info","event":"Dag name:API_POSTGRE","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.630820","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"TypeError","exc_value":"expected str, bytes or os.PathLike object, not NoneType","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/API_POSTGRE.py","lineno":96,"name":"load_csv_to_stg"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-22T16:30:46.679141Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.680680Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.681051Z","level":"info","event":"Task:<Task(PythonOperator): load_csv_to_stg>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-22T16:30:46.681324Z","level":"info","event":"Failure caused by expected str, bytes or os.PathLike object, not NoneType","chan":"stdout","logger":"task"}
